components/eamxx/src/physics/gw/impl/gw_gwd_compute_tendencies_from_stress_divergence_impl.hpp

```diff
@@ -11,6 +12,12 @@ namespace gw {
  * #include this file, but include gw_functions.hpp instead.
  */
 
+// TODO: Move to common area
+template <typename T>
+T sign(T a, T b) {
+    return (b >= 0) ? std::abs(a) : -std::abs(a);
+}
+
 template<typename S, typename D>
 KOKKOS_FUNCTION
```

Comment:
Here again I'm thinking this is not a function called from a kernel, since there are view allocs below and loops over all the cols. So `KOKKOS_FUNCTION` shouldn't be used.